src/component/personinfo/components/navnogalder.tsx

```diff
@@ -9,6 +9,9 @@ export function kalkulerAlder(fodselsdato: Date): number {
     }
     return alder;
 }
+export function Navn({ navn }: { navn: string }) {
+    return <Heading size="xsmall" level="3">{`${navn}`}</Heading>;
```

Comment:
Vi trenger ikke å lage funksjonen/component for å vise navn fordi vi manipulerer ikke navn, vi bare viser det som det er. Du kan bruke `<Heading size="xsmall" level="3">{navn}</Heading> ` direkte i huskelapp-redigere-modal. Også det holder med bare {navn} istedenfor {`${navn}`} ellers bra jobbe 👍